src/lib/storage/storage.ts

```diff
@@ -215,6 +224,15 @@ export class Storage {
     await this.connection.manager.save(encryptedSeed)
   }
 
+  private async deleteEncryptedSeed(encryptedEntropy: string): Promise<void> {
+    await this.createConnectionIfNeeded()
+    await this.connection.manager.createQueryBuilder()
+      .delete()
+      .from(MasterKeyEntity)
+      .where("encryptedEntropy = :encryptedEntropy", { encryptedEntropy })
+      .execute()
+  }
+
```

Comment:
s this still needed? Given that we are dropping the DB now.
We could keep it in case it's needed later I suppose.

Reply:
not entirely sure if it's needed, just gonna remove for now (it's straightforward to rewrite/bring back when/if we need it)